server/src/controllers/game.controller.js

```diff
@@ -375,7 +375,63 @@ class GameController {
   // Get Game By Id
   // async getById(req, res) {
   // }
+  
+  // send message in game chat
+  async message(req, res) {
+    const { message, game_id, user_id } = req.body; // user_id to be extracted from token
+    
+    // find user in db with decoded token (to be implemented later)
+    if (!user_id) {
+      throw new CustomError("Unauthorized access", 402);
+    }
+    
+    // find game in db
+    const gameExist = await GameRepo.fetchOne(game_id);
+    if (!gameExist.data) {
+      throw new CustomError("Game instance not found", 404);
+    }
 
+    // players should not be able to send message (to be implemented later)
+    
+    // incase user gets sloppy
+    const formattedMessage = message.trim();
+    if (!formattedMessage) {
+      throw new CustomError("message text cannot be empty", 400);
+    }
+
+    // test usernames and image_url
+    const testUsernames = ["mark", "jack", "jane", "crystal"];
```

Comment:
Take these test variables out na…

Reply:
leave them there for now, once there's a way to fetch the real user info i'd remove and update it